2_process.R

```diff
@@ -448,7 +454,11 @@ p2_targets_list <- list(
   # Create combined NHDv2 attribute data frame that includes both the cumulative upstream and catchment-scale values
   tar_target(
     p2_nhdv2_attr,
-    create_nhdv2_attr_table(p2_nhdv2_attr_upstream, p2_nhdv2_attr_catchment)
+    create_nhdv2_attr_table(p2_nhdv2_attr_upstream, p2_nhdv2_attr_catchment) %>%
+      #add CAT road salt to static attributes
+      left_join(p2_rdsalt_per_catchment_allyrs %>% 
+                  select(PRMS_segid, rd_salt_all_years_prop_drb) %>%
+                  rename(CAT_rdsalt_prop = rd_salt_all_years_prop_drb), by = 'PRMS_segid')
```

Comment:
Here, the road salt values are appended to the data frame that contains all of the attributes summarized from the datasets on ScienceBase. Are there other datasets that were calculated by raster processing that could also be joined here? Or is land cover the only one (and land cover you are only treating as a dynamic attribute, rather than as a static attribute)?

Reply:
I debated also joining the 2011 riparian buffer attributes, but that dataset is currently not processed to PRMS segments so I decided to omit it for now. The other land cover data do not need to be merged here because they're handled dynamically. So I don't think there are others that would need to be added here, but thanks for checking!